sqlglot/tokens.py

```diff
@@ -145,6 +146,7 @@ class TokenType(AutoName):
     DESCRIBE = auto()
     DETERMINISTIC = auto()
     DISTINCT = auto()
+    DISTINCT_FROM = auto()
```

Comment:
unsure if it's preferable to piggy back off of the `IS` or expand both operators fully here

Reply:
this is good